packages/jsii-rosetta/test/jsii/fake-assembly.ts

```diff
@@ -0,0 +1,17 @@
+import { Assembly, SchemaVersion } from '@jsii/spec';
```

Comment:
This was moved from `jsii/assemblies.test.ts`, because `rosetta.test.ts` imports this function; but the importing actually caused all tests in `jsii/assemblies.test.ts` to be run twice as a collateral.